mix.lock

```diff
@@ -4,7 +4,7 @@
   "cowlib": {:hex, :cowlib, "1.0.2", "9d769a1d062c9c3ac753096f868ca121e2730b9a377de23dec0f7e08b1df84ee", [:make], [], "hexpm"},
   "distillery": {:hex, :distillery, "1.0.0", "a866a72bf2a3a5f078f5a249017ed951acda88a760d200512f91f585d74db1ec", [:mix], [], "hexpm"},
   "domainatrex": {:hex, :domainatrex, "2.1.2", "6eef4a16e0df5fdcb13789449ff2d3c3fb0f993aa2caf139646cd5428a585d3e", [:mix], [], "hexpm"},
-  "fetch_favicon": {:hex, :fetch_favicon, "0.1.3", "4f1aad61e0a8e2377ad1bd1638b97501668bc61faf5c10836b6269687975a45b", [:mix], [{:floki, "~> 0.20.0", [hex: :floki, repo: "hexpm", optional: false]}, {:httpoison, "~> 1.0", [hex: :httpoison, repo: "hexpm", optional: false]}], "hexpm"},
+  "fetch_favicon": {:hex, :fetch_favicon, "0.1.3", "379533b622d6a62dfd054dc323b6b4f74f687db412df65867426aaf06edeb5ef", [:mix], [{:floki, "~> 0.20.0", [hex: :floki, repo: "hexpm", optional: false]}, {:httpoison, "~> 1.0", [hex: :httpoison, repo: "hexpm", optional: false]}], "hexpm"},
```

Comment:
I keep getting different hashes here. Do you know why?

Reply:
I do not. Let's merge it!